CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This is the Developer Changelog for Matomo platform developers. All changes in o
 
 The Product Changelog at **[matomo.org/changelog](https://matomo.org/changelog)** lets you see more details about any Matomo release, such as the list of new guides and FAQs, security fixes, and links to all closed issues. 
 
+## Matomo 3.13.6
+
+### New API
```

Comment:
`New API` doesn't seem to be the best headline here...

Reply:
The others didn't quite work either which is why I used New API.... will add a new category `API changes`